api/services/CaverService.js

```diff
@@ -117,6 +117,7 @@ module.exports = {
       id: caver.id,
       documents: caver.documents,
       exploredEntrances: caver.exploredEntrances,
+      grottos: caver.grottos,
```

Comment:
Like explained here https://github.com/GrottoCenter/Grottocenter3/wiki/Vocabulary the word "grotto" should not be used in new developments (except for the model and table name). You must use "organizations" instead (same thing for the MappingV1Service).